### PR TITLE
Map.diff update

### PR DESCRIPTION
**Pull request by boxxxie**
use both key paths from comparison objects in Map.diff

previously we just used the first objects key paths for comparison

### CODE REVIEW

File: lib/kitchen_sink/map.ex [hunk 1]
```diff
@@ -456,17 +456,37 @@ defmodule KitchenSink.Map do
...
   def diff(%{} = primary, %{} = secondary) do
-    primary_paths = primary |> key_paths()
+    primary_paths =
+      primary
+      |> key_paths()
+      |> MapSet.new()
+
+    paths =
+      secondary
+      |> key_paths
```
Comment (robertpeacock22) [inline lib/kitchen_sink/map.ex]: parens_plz()

**Review by robertpeacock22 — CHANGES_REQUESTED** [review]

File: lib/kitchen_sink/map.ex (hunk 1 above)
Comment (boxxxie) [reply lib/kitchen_sink/map.ex]: fixed

**Review by robertpeacock22 — COMMENTED** [review]
👍

**Review by robertpeacock22 — APPROVED** [review]

**Review by mikestok — APPROVED** [review]
LGTM

**Pull request merged**